### DISCUSSION TITLE
Percentage of charge / per diem with a MAX limit or a dollar CAP - In-Network File

### DISCUSSION THREAD

**Discussion by iGokul-G** [CMSgov/price-transparency-guide#403] [Ideas]
Do we need a new Max field to capture the MAX dollar limit of certain types of payments ex. 20 % of Billed charge Maxed at  $1000   or  $500 per diem with a MAX of $ 5000 for In-Network File with in negotiated_prices object

**Comment by angelamarie0975** [comment]
This is a great question. This actually came up for us the other day as well. 

After thinking about this, could this be a scenario that would be considered a 'capitated' arrangement? If so, that would be similar to those PMPM capitated arrangements with a fee schedule - we could include the % of Billed under 'ffs' arrangement as 'percentage' (negotiated_type) and then the same provider would also have a 'capitated' arrangement record for that specific billing_code and the rate would be the MAX. 

One problem I could see for the end users if we did this - they would query the file and get back 2 arrangements, they would need to understand how to piece together the 2 records - ie. if 2 arrangements, one % of billed and the other for Capitated - it should be assumed that the rate would be based on the % of billed up to the capitated rate listed in the Capitated arrangement. I don't know if this would work or not, but I was about to pose a similar question about this in regard to 'capitated' arrangements. Technically thinking - it is basically a 'capitated' agreement.

Thoughts?

**Comment by shaselton-usds** [comment]
Wrapping up old threads today -- extra information such as this that you feel the current schema doesn't allow for reporting can use the `additional_information` attribute.
